iothub/device/samples/PnpDeviceSamples/TemperatureController/Parameter.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using CommandLine;
+
+namespace TemperatureController
+{
+    /// <summary>
+    /// Parameters for the application supplied via command line arguments.
+    /// If the parameter is not supplied via command line args, it will look for it in environment variables.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            's',
+            "DeviceSecurityType",
+            HelpText = "(Required) The provisioning type that will be used for provisioning the device for the sample. Possible values include dps, connectionString (case-insensitive)." +
+            "\nDefaults to environment variable \"IOTHUB_DEVICE_SECURITY_TYPE\".")]
+        public string DeviceSecurityType { get; set; } = Environment.GetEnvironmentVariable("IOTHUB_DEVICE_SECURITY_TYPE");
+
+        [Option(
+            'p',
+            "PrimaryConnectionString",
```

Comment:
Just ConnectionString? It should not matter if you provide primary or secondary.

Reply:
Since this is a device sample, I don't see there being a confusion about this being a hub connection string. Also, I went with **Primary**ConnectionString since that is what we've decided to use in other samples as well, and keeping sync is imp.

Reply:
I still think being explicit is useful. I myself have sometimes stumbled because I am new to this. It would be good to be explicit which connection string we are expecting. Though this is a device sample, we know that it needs the device connection string as we have implemented it. As a new user one might not be sure. End of the day the device communicates to the IoTHub. For consistency, are we talking of samples across SDKs or just our samples? One can easily use the SecondaryConnectionString and succeed. This just feels a little misleading. Might be better to be precise rather than follow other samples.

Reply:
For this being connectionString vs deviceConnectionString - this info is described in the helptext: "The primary connection string for the device to simulate.". Does it still seem confusing?

For this being primaryConnectionString vs connectionSTring - see https://github.com/Azure-Samples/azure-iot-samples-csharp/pull/132/files#r486537250.

Reply:
I think naming is more important than text. People tend to not read things